### PR TITLE
EODHP-1004 airbus sar data adaptor integration

### PR DESCRIPTION
**Pull request by apalmer-tpzuk**
Note unit tests need fixing - PR raised now as draft for context and initial review.

- Moved from ordered-data to commercial-data subfolder
- Supply all new required inputs to adaptor
- Do not start an order if the requested item already exists in the user's workspace

### CODE REVIEW

File: resource_catalogue_fastapi/utils.py [hunk 1]
```diff
@@ -212,9 +216,14 @@ def execute_order_workflow(
             "workspace": user_workspace,
             "stac_key": stac_key,
             "workspace_bucket": workspace_bucket,
+            "workspaces_domain": workspaces_domain,
+            "env": "prod",
         }
     }
 
+    logger.info(f"Sending request to {url} with payload: {payload}")
+    logger.info(f"Headers: {headers}")
```
Comment (apalmer-tpzuk) [inline resource_catalogue_fastapi/utils.py:225]: Debug logging to remove

Comment (hcollingwood) [reply resource_catalogue_fastapi/utils.py:225]: Is logging bad?

Comment (apalmer-tpzuk) [reply resource_catalogue_fastapi/utils.py:225]: I'll keep the first line, but the headers will contain a user token and should not be logged long-term

File: resource_catalogue_fastapi/utils.py [hunk 2]
```diff
@@ -216,7 +216,7 @@ def execute_order_workflow(
             "workspace": user_workspace,
             "stac_key": stac_key,
             "workspace_bucket": workspace_bucket,
-            "workspaces_domain": workspaces_domain,
+            "workspace_domain": workspaces_domain,
```
Comment (hcollingwood) [inline resource_catalogue_fastapi/utils.py:219]: Use workspace_domain?

**Review by hcollingwood — APPROVED** [review]

**Pull request merged**